src/components/Layout/AuthModal/components/Register/Register.jsx

```diff
@@ -22,14 +22,187 @@ const advices = [
 ];
 
 export function Register() {
+
+	const [firstName, SetFirstName] = useState({name:"", error:"", state:true});
```

Comment:
why is the name of the setter capitalized?
```suggestion
	const [firstName, setFirstName] = useState({name:"", error:"", state:true});
```